src/server/game/variant1.js

```diff
@@ -217,29 +217,31 @@ const variant1 = (function() {
             case "Knightline":
                 initKnightline(gamefile, { Variant, Date })
                 break;
-            case "Knighted Chess":
+            case "Knighted_Chess":
                 initKnightedChess(gamefile, { Variant, Date })
                 break;
             // Showcasings...
             case "Omega": // Joel & Cory's version
-                variantomega1.initOmega(gamefile, { Variant, Date });
+                variantomega.initOmega(gamefile, { Variant, Date });
```

Comment:
Whoops! You accidentally referenced the wrong script! Should be `variantomega1`. This is hard to catch. For a fast way to convert them all, rename the "variantomega1" import at the top of the module to "variantomega", thenafter Right Click > Rename symbol, and rename back to "variantomega1". This will change all instances of "variantomega" in the module to "variantomega1"!

Reply:
Oh, my bad! Apparently this was wrong in a few more places in that script that I did not touch.